### ISSUE TITLE
 Vite Support

### ISSUE THREAD

**Issue by BennyAlex** [theKashey/react-imported-component#227]
@theKashey  Hello,

I want to know how to use this libary with vite? (Without SSR)

Since there is no babel I cant use the babel plugin, so is this libary not working with vite? (And therefore not bundler independent?)

Thanks, have a good day.

**Comment by theKashey** [comment]
😃 yeah, independent from a bundler, but not from babel 🤷‍♂️

In any case - babel is not required for anything NOT involving SSR, and you've said you don't have it.
The responsibility of babel is to "inline" filenames and might name webpack chunks via _magic comments_ to _communicate_ what was done during SSR and shall be redone on the client. One can easily replicate the same via rollup plugin (especially if [rollup plugin can just call a babel one](https://github.com/callstack/linaria/blob/master/packages/rollup/src/index.ts#L96))

So, let me be honest, I never tried imported with Vite. Is it working? Is it not?

**Comment by BennyAlex** [comment]
I can try it. Is there any config need than or do I just install the package and use the imports?

**Comment by theKashey** [comment]
Just using should be fine. Marks are important only for hydration phase, later they are used only as fallback - https://github.com/theKashey/react-imported-component/blob/master/src/loadable/loadable.ts#L43

**Comment by PawanKolhe** [comment]
Hey @theKashey. We are using `react-imported-component` in our app (with Vite bundler) without SSR (clIent side SPA only). Do I need to worry about the warning about no mark found in the browser console? 

![Screenshot 2024-05-10 at 8 05 16 AM](https://github.com/theKashey/react-imported-component/assets/8324407/723f5b26-e0e2-4084-8678-3209f5534970)

**Comment by theKashey** [comment]
Without SSR marks are not important and I might need to provide a way to suppress this error after hydration as it only creates noise

**Comment by PawanKolhe** [comment]
@theKashey Right. The warnings are creating a lot of noise in the console. A way to suppress them would be very helpful.
